tls/tests/test_remote.py

```diff
@@ -134,20 +134,6 @@ def test_no_connect_ipv6(aggregator, instance_remote_no_connect):
     aggregator.assert_all_metrics_covered()
 
 
-def test_version_default_1_1(aggregator, instance_remote_version_default_1_1):
```

Comment:
OpenSSL in Ubuntu 20+ dropped TLS 1.1 by default

Reply:
Should we update the tls README to mention that we only support TLS 1.2+ now?